packages/http-client-java/generator/http-client-generator-test/tsp/multiple-content-types.tsp

```diff
@@ -71,6 +73,7 @@ interface MultipleContentTypesOnRequest {
   @route("upload/multi-body-types")
   @sharedRoute
   @post
+  @convenientAPI(false)
```

Comment:
Cause by [case7](https://gist.github.com/haolingdong-msft/df3f25a048baf83d8fabf5206aacc7db#7-multiple-content-types-case-models-and-header-enum-will-be-generated-after-adopting-tcgc). We agree to add `@convenientAPI(false)` to the operation.